include/industry_standard/spdm.h

```diff
@@ -174,6 +227,8 @@ typedef struct {
     /* Below field is added in 1.2.*/
     uint32_t data_transfer_size;
     uint32_t max_spdm_msg_size;
+    /* Below field is added in 1.3.*/
+    spdm_supported_algorithms_block_t supported_algorithms;
```

Comment:
maybe just comment this field?

This size is NOT accurate anyway.

Reply:
This is consistent with the other `struct` where they expand with each SPDM version.
>This size is NOT accurate anyway.

This uses `spdm_supported_algorithms_block_t` which is accurate. `spdm_responder_supported_algorithms_t`, which is returned to the Integrator, would be inaccurate.

Reply:
> This uses spdm_supported_algorithms_block_t which is accurate.

I don't think so, the `spdm_supported_algorithms_block_t` does not include the extended algorithms and struct table, which are variable length.